.github/kokoro/presubmit/strong.cfg

```diff
@@ -8,6 +8,7 @@ timeout_mins: 60
 action {
   define_artifacts {
     # File types
+    regex: "**/*.out"
```

Comment:
Probably want this for continuous/weeky and continuous/strong too?

Reply:
Yep good point, I'll put those in too.